nixos/modules/installer/sd-card/sd-image.nix

```diff
@@ -18,12 +18,11 @@ with lib;
 let
   rootfsImage = pkgs.callPackage ../../../lib/make-ext4-fs.nix ({
     inherit (config.sdImage) storePaths;
-    compressImage = true;
     populateImageCommands = config.sdImage.populateRootCommands;
     volumeLabel = "NIXOS_SD";
   } // optionalAttrs (config.sdImage.rootPartitionUUID != null) {
     uuid = config.sdImage.rootPartitionUUID;
-  });
+  }) // config.sdImage.makeFsExtraArgs;
```

Comment:
IMHO this should be `config.sdImage.makeFsArgs // { ... } // optionalAttrs ...`. Should make it slightly harder for people to shoot themselves in the foot.

Reply:
Thanks for the lint, however, this code is not present anymore